repository/kafka/operator/templates/statefulset.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       containers:
         {{ if eq .Params.METRICS_ENABLED "true" }}
         - name: kafka-node-exporter
-          image: quay.io/prometheus/node-exporter
+          image: quay.io/prometheus/node-exporter:v0.18.1
```

Comment:
lets add this information to versions.md also

Reply:
updated :+1: